### PR TITLE
Refactor test_couchdb_session_auth.py to follow core change

### LINKED ISSUE #67: Compatibility break in `couchdb_session_auth` with moving core from 3.3.6 to 3.5.0
There seems to be a compatibility break in `couchdb_session_auth` module with moving core from v3.3.6 to v3.5.0. A test failure is consistent for all python versions and in both Travis and Jenkins:

```
[2021-01-27T13:57:19.250Z] test/unit/test_couchdb_session_auth.py ....RRRF...                       [100%]
[2021-01-27T13:57:19.250Z] 
[2021-01-27T13:57:19.250Z] =================================== FAILURES ===================================
[2021-01-27T13:57:19.250Z] ___________ TestCouchDbSessionAuth.test_disable_ssl_verification_on ____________
[2021-01-27T13:57:19.251Z] 
[2021-01-27T13:57:19.251Z] self = <test_couchdb_session_auth.TestCouchDbSessionAuth testMethod=test_disable_ssl_verification_on>
[2021-01-27T13:57:19.251Z] args = (<MagicMock name='request' id='140154731883880'>,)
[2021-01-27T13:57:19.251Z] 
[2021-01-27T13:57:19.251Z]     @responses.activate
[2021-01-27T13:57:19.251Z]     @mock.patch('requests.request', wraps=requests_request_spy)
[2021-01-27T13:57:19.251Z]     def test_disable_ssl_verification_on(self, *args):
[2021-01-27T13:57:19.251Z]         global request_kwargs
[2021-01-27T13:57:19.251Z]         self.client.set_disable_ssl_verification(True)
[2021-01-27T13:57:19.251Z]         self.client.get_session_information()
[2021-01-27T13:57:19.251Z] >       assert request_kwargs['verify'] is False
[2021-01-27T13:57:19.251Z] E       KeyError: 'verify'
```

Ref: https://github.com/IBM/cloudant-python-sdk/pull/66

**To Reproduce**
1. Change `requirements.txt` to `ibm_cloud_sdk_core==3.5.0`
2. Run `python3 -m tox -e py36`

**Expected behavior**
Tests should pass

### PR DESCRIPTION
**Pull request by bessbd**
## PR summary

This fixes #67

## PR Checklist

Please make sure that your PR fulfills the following requirements:

- [x] The commit message follows the
[Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
- [ ] Tests for the changes have been added (for bug fixes / features) - N/A
- [ ] Docs have been added / updated (for bug fixes / features) - N/A

## PR Type  

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [ ] New tests
- [ ] Build/CI related changes
- [ ] Documentation content changes
- [ ] Other (please describe)

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: test/unit/test_couchdb_session_auth.py [hunk 1]
```diff
@@ -31,11 +31,15 @@
 request_kwargs = None
 
 
-def requests_request_spy(*args, **kwargs):
```
Comment (vmatyusGitHub) [inline test/unit/test_couchdb_session_auth.py:34]: Found some unused import statements:
 `import responses`
`from unittest import mock`

**Review by vmatyusGitHub — APPROVED** [review]
LGTM, +1

File: test/unit/test_couchdb_session_auth.py (hunk 1 above)
Comment (bessbd) [reply test/unit/test_couchdb_session_auth.py:34]: Good call! I'll push a commit for this soon.

Comment (bessbd) [reply test/unit/test_couchdb_session_auth.py:34]: Done in 4fb655b

Comment (bessbd) [reply test/unit/test_couchdb_session_auth.py:34]: https://travis-ci.com/github/IBM/cloudant-python-sdk/jobs/479659186#L323 says `responses` is not unused :/

Comment (bessbd) [reply test/unit/test_couchdb_session_auth.py:34]: Re-added in 2b139034092403a86fd55d70022095425c476907

Comment (vmatyusGitHub) [reply test/unit/test_couchdb_session_auth.py:34]: > https://travis-ci.com/github/IBM/cloudant-python-sdk/jobs/479659186#L323 says responses is not unused :/

Sorry, missed that.

**Review by emlaver — APPROVED** [review]

**Comment by bessbd** [comment]
Thank you for the review, @vmatyusGitHub and @emlaver ! I'm about to merge this.

**Pull request merged**